bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js

```diff
@@ -45,19 +45,29 @@ pimcore.object.tags.advancedManyToManyObjectRelation = Class.create(pimcore.obje
         var visibleFields = Ext.isString(this.fieldConfig.visibleFields) ? this.fieldConfig.visibleFields.split(",") : [];
         this.visibleFields = visibleFields;
 
-        fields.push("id");
-        fields.push("index");
-        fields.push("inheritedFields");
-        fields.push("metadata");
+        fields.push({name: "id"});
+        fields.push({name: "index"});
+        fields.push({name: "inheritedFields"});
+        fields.push({name: "metadata"});
 
         var i;
 
         for (i = 0; i < visibleFields.length; i++) {
-            fields.push(visibleFields[i]);
+            fields.push({name: visibleFields[i]});
         }
 
         for (i = 0; i < this.fieldConfig.columns.length; i++) {
-            fields.push(this.fieldConfig.columns[i].key);
+            var defaultValue = null;
+            switch(this.fieldConfig.columns[i].type.toLowerCase()){
+                case "bool":
```

Comment:
Any reason columnbool isn't included here like in the one below? I'm not even clear on what the difference is between those

Reply:
If I remember correctly, the Object version doesnt provide it as an option

Reply:
You're correct - I thought I saw it there in my test but nope